tests/testthat/test-indentation_linter.R

```diff
@@ -163,6 +173,19 @@ test_that("function argument indentation works in tidyverse-style", {
     linter
   )
 
+  # new style (#1754)
```

Comment:
maybe a few more tests? e.g. triple-indenting gets linted.

does mixing single- and double-indenting lint?

single-indenting should lint under "tidy" too right?